src/rgw/rgw_rest_s3.cc

```diff
@@ -6552,10 +6548,12 @@ int RGWSelectObj_ObjStore_S3::send_response_data(bufferlist& bl, off_t ofs, off_
     i++;
   }
   m_aws_response_handler = std::make_unique<aws_response_handler>(s,this);
```

Comment:
its wrong to init /m_aws_response_handler/ , in more than one place.
should be init , one time.